### DISCUSSION TITLE
Upgrading to version 2.0 - Changes we must know

### DISCUSSION THREAD

**Discussion by Amtech281** [firecmsco/firecms#405] [General]
hi @fgatti675 
We would like to upgrade to version 2.0.0-alpha, however I dont have a clear overview what has changed and what are the risks.
Where can I find information what changed (and we therefore must check in advance), what might not work anymore (as I understand some changes in permissions?) and in general have an idea how much changed/stable firemcs 2.0.0 is
thanks for your feedback!

**Comment by fgatti675** [comment]
Hi @Andre3424 
You can see all the updates in https://firecms.co/docs/new_in_v2
and if you want to migrate, there is a guide for that too :)
https://firecms.co/docs/migrating_from_v1

**Reply by Amtech281** [reply]
@Andre3424 great, thanks!
